opencraft/settings.py

```diff
@@ -296,7 +296,7 @@
     'immediate': env.bool('HUEY_ALWAYS_EAGER', default=False),
 
     # Options to pass into the consumer when running ``manage.py run_huey``
-    'consumer': {'workers': 1, 'loglevel': logging.INFO},
+    'consumer': {'workers': 1, 'loglevel': logging.INFO, 'worker_type': 'thread'},
```

Comment:
@fox, can we also make this configurable?

Reply:
This is configurable-- you can always override when calling the management command by changing the args (we do this for the worker count in the procfile, though THAT should be made configurable as you've noted).

In fact this code change has no impact, it's just there to make it more explicit which worker type is being chosen, and to make sure if upstream changes the default, our default is not changed.

Reply:
Looking closer at the way we're invoking Huey in the procfile, and the way we do not access environment variables there, perhaps we SHOULD actually make it configurable here. I'll add that now.